### PR TITLE
Support free texts in Page Header, Empty State and Alert.

### LINKED ISSUE #753: Provide a way to add text with formatting in Empty State, Page Header, Alert.
Empty State, Page Header, Alert have a way to add text but no formatting is allowed. These components will need to allow text with either **bold**, [link](url) or sometimes `code`. Note that [Form field helper text](https://auth0-cosmos.now.sh/docs/#/component/form-field#help-text) has a way, but it's wonky. Maybe it can be reviewed as well.

screen shot 2018-08-02 at 16 45 49

Either way, it's interesting to check if some API decisions are shared or not.

### PR DESCRIPTION
**Pull request by francocorreasosa**
Resolves #753

### CODE REVIEW

File: core/components/_helpers/free-text.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+export function renderText(text, children, { useParagraph = false } = {}) {
+  if (children) {
+    if (useParagraph) return <Text>{children}</Text>
+    return children
+  }
+
+  if (typeof text === 'string') {
+    if (useParagraph) return <Text>{text}</Text>
+    console.log({ useParagraph })
```
Comment (nkohari) [inline core/components/_helpers/free-text.js]: I know this is still WIP, but be sure to remove this. ;)

**Comment by nkohari** [comment]
Could we handle this using an internal component called `Text`, which we'd just pass a spread of props to? For example:

```jsx
<Text {...props} />
```

The implementation of the component would be the same as the `renderText()` function, really, but the API might feel a little easier to work with. Just a thought, though, feel to disagree!

File: core/components/_helpers/free-text.js (hunk 1 above)
Comment (francocorreasosa) [reply core/components/_helpers/free-text.js]: Oops! Thanks

**Comment by francocorreasosa** [comment]
Sounds good! 👍

**Comment by francocorreasosa** [comment]
Done! API is much simpler now, thank you! @nkohari

**Comment by siddharthkp** [comment]
Oops! Did we make a breaking change in this PR?

The old API no longer works!
```jsx
<PageHeader description={{
  text: 'Setup a mobile, web or IoT application to use Auth0 for Authentication.',
  learnMore: '/clients'
}}/>
```

Sorry for not catching it earlier!

**Comment by francocorreasosa** [comment]
Hey @siddharthkp 👋 I tested it and it works.

image

**Comment by siddharthkp** [comment]
🤔

I'm I confusing this with the children version?

Check out the .story.js just to be sure

**Comment by francocorreasosa** [comment]
The children version was removed of this component. cc @landitus

**Comment by siddharthkp** [comment]
Oh, should we _deprecate_ it instead of removing it? Just, so that we can say backward compatible?

**Comment by francocorreasosa** [comment]
We did not reached a stage where the `children` version was public/released. So nobody got to use it and there is no reason to deprecate it

**Pull request merged**